packages/cadl-ranch-specs/http/encode/numeric/main.tsp

```diff
@@ -13,53 +13,60 @@ namespace Encode.Numeric;
 @operationGroup
 @route("/property")
 namespace Property {
+  alias SendSafeIntAsString = SendIntAsString<safeint, "10000000000", SafeintAsStringProperty>;
+
   @route("/safeint")
-  @scenario
-  @scenarioDoc("""
-    Test operation with request and response model contains property of safeint type with string encode.
-    Expected request body:
-    ```json
-    {
-      "value": "10000000000"
-    }
-    ```
-    Expected response body:
-    ```json
-    {
-      "value": "10000000000"
-    }
-    ```
-    """)
-  @post
-  op safeintAsString(@body body: SafeintAsStringProperty): SafeintAsStringProperty;
+  op safeintAsString is SendSafeIntAsString.sendIntAsString;
 
   model SafeintAsStringProperty {
     @encode(string)
     value: safeint;
   }
 
+  alias SendUint32AsString = SendIntAsString<uint32, "1", Uint32AsStringProperty>;
+
   @route("/uint32")
-  @scenario
-  @scenarioDoc("""
-    Test operation with request and response model contains property of uint32 type with string encode.
-    Expected request body:
-    ```json
-    {
-      "value": "1"
-    }
-    ```
-    Expected response body:
-    ```json
-    {
-      "value": "1"
-    }
-    ```
-    """)
-  @post
-  op uint32AsStringOptional(@body body: Uint32AsStringProperty): Uint32AsStringProperty;
+  op uint32AsStringOptional is SendUint32AsString.sendIntAsString;
```

Comment:
nit: we shouldn't need the alias, you should just be able to directly do `op uint32AsStringOptions is SendIntAsString<uint32, "1", Uint32AsStringProperty>`

Reply:
Looks like the compiler won't allow it.
![image](https://github.com/user-attachments/assets/0c213c41-fb69-4a9a-ac5d-c4e979467810)

I'll merge this first. We can create a separate PR to refactor if needed.